tools/editor/dependency_editor.cpp

```diff
@@ -420,7 +418,6 @@ void DependencyRemoveDialog::show(const Vector<String> &to_erase) {
 	exist=false;
 	owners->clear();
 	files.clear();
-	TreeItem *root=owners->create_item();
```

Comment:
Yes, this will modify `owners`. It's possible to get that later with `Tree::get_root()`. So you can remove the variable but not the function call.